### PR TITLE
feat(events): allow to hide tentative events #515

### PR DESCRIPTION
**Pull request by jgoldhammer**
### Status
**READY**

### Description
- allows to show, hide or mark tentative events as inactive. Tentative events are maybe events where the recipient do not know if he will attend.

## Checklist
- [ ] Localized
- [ ] Added to changelog:
  - [ ] [Changelog View](https://github.com/leits/MeetingBar/blob/master/MeetingBar/Views/Changelog/Changelog.swift)
  - [ ] [CHANGELOG.md](https://github.com/leits/MeetingBar/blob/master/CHANGELOG.md)

### Steps to Test or Reproduce
- Send an event from a second calendar provider 
- Answer event with "tentative/maybe"
- Check the menubar /statusmenu by configure the new setting

### CODE REVIEW

**Comment by jgoldhammer** [comment]
@leits - do you think this is a good way to implement it?

**Comment by jgoldhammer** [comment]
What do you think, @leits?

**Comment by jgoldhammer** [comment]
@leits - what do you think?

**Comment by leits** [comment]
LGTM, sorry for the delay!

File: MeetingBar/StatusBarItemController.swift [hunk 1]
```diff
@@ -467,6 +485,13 @@ class StatusBarItemController {
             } else {
                 styles[NSAttributedString.Key.font] = NSFont.systemFont(ofSize: 14)
             }
+            
+            if shouldShowAsActive, Defaults[.showTentativeEvents] != TentativeEventsAppereance.show_underlined {
+                // add the NSTextAttachment wrapper to our full string, then add some more text.
+                styles[NSAttributedString.Key.font] = NSFont.boldSystemFont(ofSize: 14)
```
Comment (leits) [inline MeetingBar/StatusBarItemController.swift:491]: bold?

**Comment by joergsti** [comment]
Any news on this PR? This would dramatically boost the benefit of MeetingBar for my scenario.

**Review by leits — APPROVED** [review]

**Pull request merged**